api/src/main/java/org/openmrs/api/db/ClobDatatypeStorage.java

```diff
@@ -11,17 +11,29 @@
 
 import org.openmrs.BaseOpenmrsObject;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 /**
  * Stores a potentially-long String value, for clob-based custom datatypes
  * 
  * @since 1.9
  */
+@Entity
+@Table(name = "clob_datatype_storage")
 public class ClobDatatypeStorage extends BaseOpenmrsObject {
 	
+	@Id
+	@Column(name = "clob_datatype_storage_id")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Integer id;
 	
 	// inherits uuid from BaseOpenmrsObject
-	
+	@Column(name = "value")
 	String value;
```

Comment:
@alinmihaila99, you will have to use JPA's `@Lob` for this to work because of this [underlying liquibase changeset](https://github.com/openmrs/openmrs-core/blob/e3b1d06cc494a60617a2304412b8d47b89b5a2b0/api/src/main/resources/org/openmrs/liquibase/snapshots/schema-only/liquibase-schema-only-2.3.x.xml#L111).

Reply:
Thanks @Ruhanga. This worked